tests/hats/catalog/dataset/test_table_properties.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from hats.catalog.dataset.table_properties import TableProperties
+
+
+@pytest.mark.parametrize("data_dir", ["catalog", "dataset", "index_catalog", "margin_cache"])
```

Comment:
```suggestion
@pytest.mark.parameterize("data_dir", ["catalog", "dataset", "index_catalog", "margin_cache"])
```

This looks like a typo to me.  Does the decorator itself have this typo?

Reply:
That's the name!

https://docs.pytest.org/en/7.1.x/how-to/parametrize.html

https://en.wikipedia.org/wiki/Parametrization